reference/forms/types/birthday.rst

```diff
@@ -67,7 +67,30 @@ type:
 
 .. include:: /reference/forms/types/options/days.rst.inc
 
-.. include:: /reference/forms/types/options/placeholder.rst.inc
+placeholder
+~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``placeholder`` option was introduced in Symfony 2.6 and replaces
+    ``empty_value``, which is available prior to 2.6.
+
+**type**: ``string`` or ``array``
+
+If your widget option is set to ``choice``, then this field will be represented
+as a series of ``select`` boxes. The ``placeholder`` option can be used
+to add a "blank" entry to the top of each select box::
```

Comment:
I would put this a bit more generic saying something like "when the placeholder is a string, it will be used as the **blank value** of all select boxes: [....] Alternatively, you can use an array that configures different placeholder values for the day, month and year fields."